typings/react-datetime-tests.tsx

```diff
@@ -0,0 +1,170 @@
+import * as React from 'react';
+import { Moment } from 'moment';
```

Comment:
This file is written with ES6 syntax. I prefer ES6 syntax, but none of the other files are written in this syntax. My first thought is that we should stay consistent, but at the same time this could be the first step to making everything into ES6 (which I plan on doing eventually). What are your thoughts on this?